test/conftest.py

```diff
@@ -1,9 +1,14 @@
 # Copyright 2022-present ScyllaDB
 #
 # SPDX-License-Identifier: AGPL-3.0-or-later
-
-# This file configures pytest for all tests in this directory, and also
-# defines common test fixtures for all of them to use.
+#
+# Common pytest fixtures for *all* tests in this directory.
+#
+# Please avoid adding bloat (you can also import
+# fixtures directly from test/pylib).
+#
+# To enable, add addopts = --confcutdir .. to your pytest.ini
+#
```

Comment:
Oh, here is the comment I asked for...
The first and last lines ("*all* tests) and "to enable, ...", are contradictory. It's not all tests if it's only those which enable it. Maybe better say something like:

Common pytest fixtures for all subdirectories which specifically request these
common pytest fixtures by adding "addopts = --confcutdir .." in their pytest.ini.